templates/haproxy.cfg.j2

```diff
@@ -1,10 +1,9 @@
 global
-    maxconn 4096
+    maxconn {{ config_global_max_connection }}
     user haproxy
     group haproxy
     
-    log /dev/log local0
-    log /dev/log local1 notice
+    log 127.0.0.1:514 local0
```

Comment:
Should we add a comment to explain why we set this valye ?

Reply:
Added the comment about rsyslog